### PR TITLE
[fr] Translating docs/reference/access-authn-authz/authentication.md

### PR DESCRIPTION
**Pull request by skurtzemann**
French translation of [Reference/API Access Control/Authenticating](https://kubernetes.io/docs/reference/access-authn-authz/authentication/) page

### CODE REVIEW

File: content/fr/docs/reference/access-authn-authz/authentication.md [hunk 1]
```diff
@@ -0,0 +1,1244 @@
...
+<!-- body -->
+## Utilisateurs dans Kubernetes
+
+Tous les clusters Kubernetes possèdent deux catégories d'utilisateurs : les comptes de services managés par Kubernetes et les utilisateurs normaux.
+
+On suppose qu'un service indépendant du cluster gère les utilisateurs normaux de la façon suivante :
+
+- un administateur est en charge de distribuer les clés privées
+- un (service d'identité)/registre d'utilisateur comme (OpenStack) Keystone and Google Accounts
+- un fichier contenant une liste des utilisateurs et des mots de passes
```
Comment (perriea) [inline content/fr/docs/reference/access-authn-authz/authentication.md:26]: `passe` ne s'accorde pas 
```suggestion
- un fichier contenant une liste des utilisateurs et des mots de passe
```

**Comment by divya-mohan0209** [comment]
Hello @kubernetes/sig-docs-fr-owners & @kubernetes/sig-docs-fr-reviews, PTAL. I am not trying to overstep and impose here and I understand that the cadence for review and approvals within localization subgroups is different. However, if the lack of approvers/reviewers is a challenge and merits a discussion in the localization subproject meeting, please voice out. Alternately, please feel free to reach out to the localization or docs leads privately if you're unable or aren't comfortable bringing it up in the meeting.

**Comment by rekcah78** [comment]
@divya-mohan0209 this page is still in 'Draft' status, we are waiting for an update from the contributor, a large part is not yet translated.

**Comment by divya-mohan0209** [comment]
Hello @rekcah78 , thank you for the update! . We understand that things work differently when it comes to a localisation, being a localisation maintainer myself, however this has been open since the last year without any action being taken on it. I'd be closing this off right now, keeping in mind, PR hygiene & project health. If the submitter wishes to pick up from where they left off, they can reopen the PR by issuing the /reopen command.

/close

**Pull request closed without merging**